src/families/polkadot/NominateFlow/01-Validators.js

```diff
@@ -0,0 +1,320 @@
+    transaction.validators,
+  ]);
+
+  const nominations = useMemo(() => polkadotResources.nominations || [], [
+    polkadotResources.nominations,
+  ]);
+
+  const { validators: polkadotValidators } = usePolkadotPreloadData();
+  const sorted = useSortedValidators(
+    searchQuery,
+    polkadotValidators,
+    nominations,
+  );
+
+  const sections = useMemo(
+    () =>
+      sorted
+        .reduce(
+          (data, validator) => {
+            const isNominated = nominations.some(
+              n => n.address === validator.address,
+            );
+            if (isNominated) {
+              data[0].data.push(validator);
+            } else if (validator.isElected) {
+              data[1].data.push(validator);
+            } else {
+              data[2].data.push(validator);
+            }
+            return data;
+          },
+          [
+            {
+              title: (
+                <Trans i18nKey="polkadot.nominate.steps.validators.myNominations" />
+              ),
+              data: [],
+            },
+            {
+              title: (
+                <Trans i18nKey="polkadot.nominate.steps.validators.electedValidators" />
+              ),
+              data: [],
+            },
+            {
+              title: (
+                <Trans i18nKey="polkadot.nominate.steps.validators.waitingValidators" />
+              ),
+              data: [],
+            },
+          ],
+        )
+        .filter(({ data }) => data.length > 0),
+    [sorted, nominations],
+  );
+
+  const onNext = useCallback(() => {
+    navigation.navigate(ScreenName.PolkadotNominateSelectDevice, {
+      ...route.params,
+      transaction,
+      status,
+    });
+  }, [navigation, route.params, transaction, status]);
+
+  const onSelect = useCallback(
+    (validator, selected) => {
+      const newValidators = selected
+        ? validators.filter(v => v !== validator.address)
+        : [...validators, validator.address];
+      const tx = bridge.updateTransaction(transaction, {
+        validators: newValidators,
+      });
+      setTransaction(tx);
+    },
+    [bridge, setTransaction, transaction, validators],
+  );
+
+  const onOpenExplorer = useCallback(
+    (address: string) => {
+      const url = getAddressExplorer(
+        getDefaultExplorerView(mainAccount.currency),
+        address,
+      );
+      if (url) Linking.openURL(url);
+    },
+    [mainAccount.currency],
+  );
+
+  const renderItem = useCallback(
+    ({ item }) => {
+      const selected = validators.indexOf(item.address) > -1;
+
+      const disabled = validators.length >= MAX_NOMINATIONS;
+
+      return (
+        <ValidatorItem
+          item={item}
+          disabled={disabled}
+          selected={selected}
+          onSelect={onSelect}
+          onOpenExplorer={onOpenExplorer}
+        />
+      );
+    },
+    [validators, onSelect, onOpenExplorer],
+  );
+
+  const error = status && status.errors && Object.values(status.errors)[0];
```

Comment:
Yes, in the bottom, above the Continue button.
But there is some work to do, since we planned to ignore the error if no validator is selected, and display a context-dependent message to redirect user to the Chill Flow (or just ignore the error if it's his first nominate, to avoid displaying an error when starting to nominate).